### PR TITLE
added qwen1.5-7b to model list

### PR DESCRIPTION
**Pull request by mengbingrock**
I've verified support of Qwen1.5-7B by OpenVINO and then added it to the github workflow and readme.md
```
(base) root@8tvt:~/openvino.genai/llm_bench/python# ../../text_generation/causal_lm/cpp/build/greedy_causal_lm qwen/pytorch/dldt/FP32/ "Why is the Sun yellow?"

 

The Sun does not actually appear yellow to us when we look at it. In fact, it appears white because it emits light across a wide range of wavelengths, including all the colors of the visible spectrum. When this light reaches our eyes, our eyes combine the different colors to create the perception of white.
```

### CODE REVIEW

**Comment by p-wysocki** [comment]
cc @Wovchena @pavel-esir

File: text_generation/causal_lm/cpp/README.md [hunk 1]
```diff
@@ -127,9 +127,11 @@ To enable Unicode characters for Windows cmd open `Region` settings from `Contro
    4. https://huggingface.co/openlm-research/open_llama_7b
    5. https://huggingface.co/openlm-research/open_llama_7b_v2
 5. [TinyLlama](https://huggingface.co/TinyLlama/TinyLlama-1.1B-Chat-v1.0)
-6. Qwen
+6. Qwen and Qwen1.5
```
Comment (pavel-esir) [inline text_generation/causal_lm/cpp/README.md:130]:
```suggestion
6. Qwen
```

**Review by pavel-esir — APPROVED** [review]
looks good to me. thx @mengbingrock

File: .github/workflows/causal_lm_cpp.yml [hunk 2]
```diff
@@ -192,6 +192,32 @@ jobs:
...
+          source ./ov/setupvars.sh
+          python -m pip install --upgrade-strategy eager "optimum>=1.14" -r ./llm_bench/python/requirements.txt ./thirdparty/openvino_tokenizers/[transformers] --extra-index-url https://download.pytorch.org/whl/cpu && python ./llm_bench/python/convert.py --model_id Qwen/Qwen1.5-7B-Chat --output_dir ./Qwen1.5-7B-Chat/ --precision FP16 &
+          cmake -DCMAKE_BUILD_TYPE=Release -S ./text_generation/causal_lm/cpp/ -B ./build/
+          cmake --build ./build/ --config Release -j
+          wait
+      - name: Run
+        run: |
+          source ./ov/setupvars.sh
+          convert_tokenizer ./Qwen1.5-7B-Chat/pytorch/dldt/FP16/ --output ./Qwen1.5-7B-Chat/pytorch/dldt/FP16/ --with-detokenizer --trust-remote-code
+          timeout 50s ./build/beam_search_causal_lm ./Qwen1.5-7B-Chat/pytorch/dldt/FP16/ "你好！" > ./pred_qwen15.txt
```
Comment (ilya-lavrenov) [inline .github/workflows/causal_lm_cpp.yml:220]: could you please also add comparison with Hugging Face?
Example is https://github.com/openvinotoolkit/openvino.genai/blob/f973f6287ed5d9decf1bd3e58e88e6c813e24378/.github/workflows/causal_lm_cpp.yml#L67-L80

**Pull request merged**